doc/modules/ROOT/pages/quickstart.adoc

```diff
@@ -0,0 +1,178 @@
+there are some inconsistencies even within Microsoft's own packages that it is not possible to avoid redirects entirely.
+You can enable `<AutoGenerateBindingRedirects>` in your project file, and Visual Studio should populate your
+application config files with the appropriate binding redirects. The following redirects should be sufficient at the moment:
++
+[source,xml]
+----
+<assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+  <dependentAssembly>
+    <assemblyIdentity name="System.Buffers" publicKeyToken="cc7b13ffcd2ddd51" culture="neutral" />
+    <bindingRedirect oldVersion="0.0.0.0-4.0.3.0" newVersion="4.0.3.0" />
+  </dependentAssembly>
+</assemblyBinding>
+<assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+  <dependentAssembly>
+    <assemblyIdentity name="System.Runtime.CompilerServices.Unsafe" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
+   <bindingRedirect oldVersion="0.0.0.0-5.0.0.0" newVersion="5.0.0.0" />
+  </dependentAssembly>
+</assemblyBinding>
+----
+====
+
+
+== Step 2. Start a Hazelcast Cluster
+
+Let's use Docker for this tutorial by executing the following command on a terminal:
+
+[source,shell,subs="attributes+"]
+----
+docker run -it --rm hazelcast/hazelcast:{server-full-version}
+----
+
+This command starts a single-server Hazelcast cluster with default settings on your local machine.
+You should see the server information in the terminal logs similar to the following (note the IP address and port to be used in a later step):
+
+[source,shell]
+----
+Members {size:1, ver:1} [
+	Member [172.17.0.2]:5701 - 603ec325-41ff-413a-8538-896972ba28c2 this
+]
+----
+
+[NOTE]
+====
+As an alternative to Docker, you can also start a Hazelcast cluster using either of the following ways:
+
+* xref:cloud:getting-started.adoc[Hazelcast Cloud]
+* xref:hazelcast:getting-started:quickstart.adoc[Hazelcast CLI]
+* xref:hazelcast:getting-started:get-started-binary.adoc[Binary distributions]
+
+If you want to use Hazelcast Cloud, follow xref:cloud:net-client.adoc[these instructions].
+
+Alternatively, the Hazelcast .NET client https://github.com/hazelcast/hazelcast-csharp-client[repository] on GitHub provides a
+https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7.2[PowerShell script] which can be used to download and run a Hazelcast test server. For instance, the following downloads and starts a server version {server-full-version}:
+
+[source,shell,subs="attributes+"]
+----
+PS> ./hz.ps1 run-server -server {server-full-version}
+----
+====
+
+== Step 3. Connect to the Cluster
+
+Once the client is installed and a default cluster is started, you're ready to use the Hazelcast client in your .NET application.
+Connect your client to the cluster with this single line:
+
+[source,csharp]
+----
+await using var client = await HazelcastClientFactory.StartNewClientAsync();
```

Comment:
We have deprecated the `StartNewClientAsync()` with no options overload. 

This could be the fastest start.
```suggestion
var options = new HazelcastOptions();
options.Networking.Addresses.Add("127.0.0.1:5701");
var client = await HazelcastClientFactory.StartNewClientAsync(options);
```